### PR TITLE
fix(make): current_project will be executed only when the relevant command is executed

### PR DESCRIPTION
**Pull request by aimuz**
Signed-off-by: aimuz <mr.imuz@gmail.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, press enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind breaking
> /kind bug
> /kind cleanup
> /kind documentation
> /kind feature
> /kind hotfix

/kind bug

**What this PR does / Why we need it**:

Before restoration

```
➜  build git:(main) make        
Unable to find image 'agones-build:f1d1057367' locally
docker: Error response from daemon: pull access denied for agones-build, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
kubectl port-forward deployments/agones-controller -n agones-system 8080
Error from server (NotFound): namespaces "agones-system" not found
make: *** [controller-portforward] Error 1

```

After restoration

```
➜  build git:(fix-make) make     
kubectl port-forward deployments/agones-controller -n agones-system 8080
Error from server (NotFound): namespaces "agones-system" not found
make: *** [controller-portforward] Error 1

```

docker run was executed in the wrong phase 

**Which issue(s) this PR fixes**:

Closes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by zmerlynn** [comment]
Thanks for the PRs! I'm waiting until after feature freeze lifts tomorrow to merge this one as well.

**Review by gongmax — APPROVED** [review]

**Pull request merged**
